modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase

```diff
@@ -170,12 +170,105 @@ definition {
 	}
 
 	@description = "Verify that a calendar event on draft can be approved"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CalendarWithDraftStatusCanBecomeApproved {
+		Navigator.gotoPage(pageName = "Calendar Page");
+
+		CalendarNavigator.gotoAddEvent();
+
+		eventTitle = "Test Event";
```

Comment:
Hi @PauloVAS, It looks like this macro is incomplete, would you mind fixing this?